tools/api-markdown-documenter/src/rendering/helpers/index.ts

```diff
@@ -10,5 +10,52 @@
  * own rendering policies.
  */
 
-export * from "./RenderingHelpers";
-export * from "./TableRenderingHelpers";
+export {
+    ChildSectionProperties,
+    DocExampleProperties,
+    renderBetaAlert,
+    renderBreadcrumb,
+    renderChildDetailsSection,
+    renderChildrenUnderHeading,
+    renderDeprecationNoticeSection,
+    renderExampleSection,
+    renderExamplesSection,
+    renderExcerptWithHyperlinks,
+    renderHeading,
+    renderHeadingForApiItem,
+    renderHeritageTypes,
+    renderLink,
+    renderParametersSection,
+    renderRemarksSection,
+    renderReturnsSection,
+    renderSeeAlso,
+    renderSignature,
+    renderSummarySection,
+    renderThrowsSection,
+    renderTypeParameters,
+} from "./RenderingHelpers";
+export {
+    MemberTableProperties,
```

Comment:
ooo @alexvy86 actually look it's because MemberTableProperties is the first export here, after a bunch of r's from the module prior.